### PR TITLE
[DEPENDS] UPDATE TO REMOVE CCACHE

### PR DESCRIPTION
**Pull request by ghost**
### [DEPENDS] UPDATE TO REMOVE CCACHE

CREDIT- [MarcoFalke] @ Bitcoin

This is port from Bitcoin allowing removal of ccache altogether from depends. Which cross-compiling for windows on linux was giving a problem with Travis.

ref: https://github.com/bitcoin/bitcoin/commit/29fad97c320c892ab6a480c81e2078ec22ab354b

### CODE REVIEW

**Comment by sedited** [comment]
Please add the original author of the commit (Marko) as the commit author.

**Comment by ghost** [comment]
This good now @TheCharlatan ?

File: .travis.yml [hunk 1]
```diff
@@ -3,10 +3,11 @@ dist: trusty
 os: linux
 language: minimal
 cache:
+  ccache: true
   directories:
   - depends/built
   - depends/sdk-sources
-  - $HOME/.ccache
```
Comment (sedited) [inline .travis.yml]: Keep this.

**Review by sedited — COMMENTED** [review]
Also reflect the change in the docs where it appears.

**Comment by ghost** [comment]
I saw no reference to this in the doc folder. Where do you see it appear?

**Pull request closed without merging**